packages/google_mlkit_language_id/lib/src/language_identifier.dart

```diff
@@ -23,9 +23,10 @@ class LanguageIdentifier {
   /// If no language could be determined then [undeterminedLanguageCode] is returned.
   /// More information: https://developers.google.com/ml-kit/language/identification
   Future<String> identifyLanguage(String text) async {
-    final result = await _channel.invokeMethod(
-        'nlp#startLanguageIdentifier', <String, dynamic>{
+    final result = await _channel
+        .invokeMethod('nlp#startLanguageIdentifier', <String, dynamic>{
       'text': text,
+      'id': id,
```

Comment:
could you please add `id` at the end of the map, just as it is in `identifyPossibleLanguages`, for the sake of having same estructure... cosmetic and readability